atom/browser/api/lib/auto-updater/auto-updater-win.js

```diff
@@ -2,70 +2,62 @@ const app = require('electron').app;
 const EventEmitter = require('events').EventEmitter;
 const url = require('url');
 const squirrelUpdate = require('./squirrel-update-win');
+const util = require('util');
 
-var extend = function(child, parent) { for (var key in parent) { if (hasProp.call(parent, key)) child[key] = parent[key]; } function ctor() { this.constructor = child; } ctor.prototype = parent.prototype; child.prototype = new ctor(); child.__super__ = parent.prototype; return child; },
-hasProp = {}.hasOwnProperty;
+function AutoUpdater() {
+  AutoUpdater.super_.call(this);
```

Comment:
In the nodejs code base, they initialize the superConstructor explicitly, instead of using `constructor.super_`. So in this case it would be `EventEmitter.call(this);`.
It's a matter of preference, but IMO it makes the intention clearer.

References:
https://nodejs.org/api/util.html#util_util_inherits_constructor_superconstructor
https://github.com/nodejs/node/search?utf8=%E2%9C%93&q=util.inherits